### DISCUSSION TITLE
Find another check for merger

### DISCUSSION THREAD

**Discussion by HannahFromAland** [jtr13/cc21fall2#149] [General]
Hi everyone, I came across a new *checkpoint* this morning when merging the #56.
The chapter name (the single hashtag line in `.Rmd` file, aka first line) cannot have `.` in it.

For example, in #56 the original chapter name was shown as:![](https://i.loli.net/2021/11/06/LDjlPfGSOJbT9ok.png)
but the workflow failed ![](https://i.loli.net/2021/11/06/V5hIZD47n1Ctkg3.png)
It worked after I revise the `data.table` into `datatable`. (since the chapter name will also be web name which will have `.html` as ending, using `.` in name will cause error)

Maybe it can be added into our tutorials as a `checkpoint`

**Comment by jtr13** [comment]
Good sleuthing... I haven't seen that before. Perhaps we can add a second heading to include data.table -- since it's the package name we can't really change it

`# Different methods for handling data in R`

`## A comparison of base R, tidyverse, and data.table`
